GHzDACs/Cleanup/dac.py

```diff
@@ -1,15 +1,72 @@
+""" dac.py
+
+This module contains the DAC class and its subclasses. DAC inherits from
+fpga.FPGA and ultimately from LabRAD's DeviceWrapper. Thus, in the main fpga
+server, when you do getDevice, you are ultimately getting one of these.
+
+Which one you get depends on the boards build number. Quick reference:
+
+* 7: build number for memory boards (i.e. pre-jump table), 8 us SRAM.
+* 8: memory boards, 16 us SRAM.
+* 13: First of the JT boards, 8 us SRAM (buggy).
+* 14: JT board, 16 us SRAM (buggy).
+* 15: JT board with ramp functionality, 16 us SRAM.
+
+Typically we use either build 8 or 15. If you add a class for a new build,
+add it to the `fpga.REGISTRY` dict (see how it's done below). If you see an
+error when starting the fpga server that says: `There was an error: ('DAC', 16)`
+this means that the fpga server found a board with build number 16 but it is
+not in the registry dict.
+
+"""
+
+import logging
 import numpy as np
+from twisted.internet.defer import inlineCallbacks, returnValue
+from labrad import types as T
 
+from servers.GHzDACs.util import littleEndian
 import servers.GHzDACs.Cleanup.fpga as fpga
+import servers.GHzDACs.jump_table as jump_table
 
-# named functions
-from twisted.internet.defer import inlineCallbacks, returnValue
+MONITOR_0 = 5
+MONITOR_1 = 10
 
-from labrad.devices import DeviceWrapper
-from labrad import types as T
+'''
+Output Monitors
+
+When dipswitch DSW[7]=1:
+Mon[0] = regdone = received Ethernet input for a register write
+Mon[1] = ethersend  =  sent Ethernet output initiated by a register readback
 
-from servers.GHzDACs.util import littleEndian, TimedLock
+When DSW[7]=0:
+Outputs of Mon[0] and Mon[1] are controlled by registers MonA[7..0] and MonB[7..0]  according to the following table
 
+0	regdone		Ethernet in, write to registers
+1	SRAMwrite		Ethernet in, write to SRAM memory
```

Comment:
Shouldn't this whole monitor documentation business live closer to the actual jump table code?